.github/workflows/voiceover-test.yml

```diff
@@ -0,0 +1,143 @@
+name: voiceover-test
+on:
+  workflow_dispatch:
+    inputs:
+      aria_at_ref:
+        description: |
+          The sha / ref to checkout for aria-at tests
+        required: false
+        type: string
+      work_dir:
+        description: |
+          The --plan-workingdir passed to the aria-at-automation harness.
+          Based from the aria-at repo build folder.
+          Uses a default of tests/alert if not provided.
+        required: false
+        type: string
+      callback_url:
+        description: |
+          The harness will send POST requests to this url using callback_header
+          header with test results.
+        required: false
+        type: string
+      status_url:
+        description: |
+          The harness will send POST requests to this url using callback_header
+          header with status updates.
+        required: false
+        type: string
+      callback_header:
+        description: |
+          The harness will send POST requests to callback_url using this header
+          with status updates.
+        required: false
+        type: string
+
+env:
+  ARIA_AT_WORK_DIR: ${{ inputs.work_dir || 'tests/alert' }}
+  ARIA_AT_CALLBACK_URL: ${{ inputs.callback_url }}
+  ARIA_AT_STATUS_URL: ${{ inputs.status_url }}
+  ARIA_AT_CALLBACK_HEADER: ${{ inputs.callback_header || 'x-header-param:empty' }}
+
+jobs:
+  voiceover-test:
+    runs-on: macos-13
+    steps:
+      - name: Log job state QUEUED
+        if: inputs.status_url
+        run: ./report-status.sh QUEUED ''
+
+      # Checkout all repos first (helps cache purposes)
+      - uses: actions/checkout@v4
+
+      - uses: robinraju/release-downloader@v1.10
+        with:
+          repository: 'w3c/aria-at-automation-driver'
+          tag: 'v0.0.1-beta'
+          extract: true
+          out-file-path: aria-at-automation-driver
+
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18.x
+          cache: "npm"
+          cache-dependency-path: "**/package-lock.json"
+
+      - name: Setup Environment
+        uses: guidepup/setup-action@0.15.3
+
+      - name: "automation-driver: npm install"
+        working-directory: aria-at-automation-driver/package
+        run: npm install
+
+      - name: "automation-driver: install"
+        env:
+          DEBUG: '*'
+        working-directory: aria-at-automation-driver/package
+        run: ./bin/at-driver install --unattended
+
+      - name: Wait for server to be ready
+        working-directory: aria-at-automation-driver/package
+        run: ../../wait-for-server.sh
+
+      - name: Checkout aria-at ref ${{ inputs.aria_at_ref || 'master' }}
+        uses: actions/checkout@v4
+        with:
+          repository: "w3c/aria-at"
+          path: "aria-at"
+
+      - name: Checkout aria-at-automation-harness
+        uses: actions/checkout@v4
+        with:
+          repository: "w3c/aria-at-automation-harness"
+          ref: safari
```

Comment:
Accidentally leftover? Is there a note that this will need a follow up PR to remove this once https://github.com/w3c/aria-at-automation-harness/pull/50 lands?

Reply:
Good idea; see [65c2805](https://github.com/bocoup/aria-at-gh-actions-helper/pull/15/commits/65c28050fce18e456ace5e19549ce6c1ef1a5d0f)